search-commons/src/test/java/no/unit/nva/search/OpensearchTest.java

```diff
@@ -209,7 +209,9 @@ void beforeEachTest() throws IOException {
         @Test
         void shouldReturnCorrectAggregationsForImportCandidates()
             throws InterruptedException, ApiGatewayException {
-            addDocumentsToIndex("imported_candidate_from_index.json", "not_imported_candidate_from_index.json");
+            addDocumentsToIndex("imported_candidate_from_index.json",
```

Comment:
I am being lazy and not looking at the code, but I'm assuming the test assertion actually corresponds with expectations that we actually have :)